### PR TITLE
`Response::headers`

### PR DESCRIPTION
**Pull request by fosskers**
This PR adds a few methods and trait instances involving headers.

### CODE REVIEW

File: src/browser/fetch/header.rs [hunk 1]
```diff
@@ -31,6 +31,24 @@ impl<'a> IntoIterator for Headers<'a> {
...
 }
 
+impl<'a> From<web_sys::Headers> for Headers<'a> {
+    fn from(hs: web_sys::Headers) -> Self {
+        let mut headers = Headers::default();
+
+        for pair in js_sys::Object::entries(hs.as_ref()).entries() {
+            if let Some((h, v)) = pair
+                .ok()
+                .and_then(|js| js.into_serde::<(String, String)>().ok())
```
Comment (fosskers) [inline src/browser/fetch/header.rs:41]: Testing this conversion now.

File: src/browser/fetch/response.rs [hunk 2]
```diff
@@ -101,6 +101,11 @@ impl Response {
     pub const fn raw_response(&self) -> &web_sys::Response {
         &self.raw_response
     }
+
+    /// Get the [`Headers`] associated with the `Response`.
+    pub fn headers(&self) -> Headers {
+        Headers::from(self.raw_response.headers())
```
Comment (fosskers) [inline src/browser/fetch/response.rs:107]: Should be a perfect conversion, hence `From` and not `TryFrom` is appropriate.

**Comment by fosskers** [comment]
See also: https://github.com/rustwasm/wasm-bindgen/issues/2572

**Comment by MartinKavik** [comment]
Let me know when it's ready to review & merge or when you need some help with the implementation :)

**Review by MartinKavik — APPROVED** [review]

**Pull request merged**